src/mono/System.Private.CoreLib/src/System/Reflection/RuntimeModule.cs

```diff
@@ -370,9 +370,9 @@ internal static Exception resolve_token_exception(Module module, int metadataTok
         internal static Exception resolve_token_exception(string name, int metadataToken, ResolveTokenError error, string tokenType)
         {
             if (error == ResolveTokenError.OutOfRange)
-                return new ArgumentOutOfRangeException(nameof(metadataToken), string.Format("Token 0x{0:x} is not valid in the scope of module {1}", metadataToken, name));
+                return new ArgumentOutOfRangeException(nameof(metadataToken), SR.Format("Token 0x{0:x} is not valid in the scope of module {1}", metadataToken, name));
```

Comment:
This should either use `string.Format` or the message should be moved into `Strings.resx` (throughout this file).